cli_tools_e2e_test/diskinspect/disk_inspect_test.go

```diff
@@ -0,0 +1,150 @@
+//  Copyright 2020 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package diskinspect
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/api/compute/v1"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/disk"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/daisycommon"
+	daisycompute "github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute"
+)
+
+var (
+	project = getConfig("GOOGLE_CLOUD_PROJECT", "project")
+	zone    = getConfig("GOOGLE_CLOUD_ZONE", "compute/zone")
+
+	wfAttrs = daisycommon.WorkflowAttributes{
+		Project:           project,
+		Zone:              zone,
+		WorkflowDirectory: "../../daisy_workflows",
+	}
+)
+
+func TestBootInspect(t *testing.T) {
+
+	client, err := daisycompute.NewClient(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for _, tt := range []struct {
+		imageURI string
+		expected disk.InspectionResult
+	}{
+		{
+			"projects/opensuse-cloud/global/images/opensuse-leap-15-2-v20200702",
+			disk.InspectionResult{
+				Architecture: "x64",
+				Distro:       "opensuse",
+				Major:        "15",
+				Minor:        "2",
+			},
+		}, {
+			"projects/suse-sap-cloud/global/images/sles-15-sp1-sap-v20200803",
+			disk.InspectionResult{
+				Architecture: "x64",
+				Distro:       "sles-sap",
+				Major:        "15",
+				Minor:        "1",
+			},
+		}, {
+			"projects/compute-image-tools-test/global/images/windows-7-ent-x86-nodrivers",
+			disk.InspectionResult{
+				Architecture: "x86",
+				Distro:       "windows",
+				Major:        "6",
+				Minor:        "1",
+			},
+		},
+	} {
+		// Without this, each parallel test will reference the last tt instance.
+		// https://github.com/golang/go/wiki/CommonMistakes#using-goroutines-on-loop-iterator-variables
+		tt := tt
```

Comment:
minor: even I know we need a local var for this purpose, I am still a bit confused here. A different var name (currentTest?) can be better.

Reply:
done!